spec/dealership_spec.rb

```diff
@@ -0,0 +1,107 @@
+require './lib/car'
+require './lib/dealership'
+
+RSpec.describe Dealership do
+  # iteration 2
+  it 'exists' do
+    dealership = Dealership.new('Acme Auto', '123 Main Street')
+    expect(dealership).to be_a(Dealership)
+  end
+
+  it 'has an inventory' do
+    dealership = Dealership.new('Acme Auto', '123 Main Street')
+    expect(dealership.inventory).to eq([])
+  end
+
+  it 'has an inventory count' do
+    dealership = Dealership.new('Acme Auto', '123 Main Street')
+    expect(dealership.inventory_count).to eq(0)
+  end
+
+  it 'has an new inventory and count ' do
+    dealership = Dealership.new('Acme Auto', '123 Main Street')
+    car_1 = Car.new('Ford Mustang', 1500, 36)
+    car_2 = Car.new('Toyota Prius', 1000, 48)
+    dealership.add_car(car_1)
+    dealership.add_car(car_2)
+    expect(dealership.inventory).to eq([car_1, car_2])
+    expect(dealership.inventory_count).to eq(2)
+  end
+
+  # Iteration 3
+  it 'has an inventory' do
+    dealership = Dealership.new('Acme Auto', '123 Main Street')
+    expect(dealership.has_inventory?).to be false
```

Comment:
Your `has_inventory?` method will always return `false` or `nil`, because of the way it's written. See the comment I made in the `Car` class file. 

To really test this functionality fully, make a test for both paths this method could return, both true and false. For false, this test is fine (except for the name, this one should be "it does not have an inventory yet") for true, something like this: 
```
it 'has an inventory' do
     dealership = Dealership.new(...)
     dealership.add_car(car_1)
     expect(dealership.has_inventory/).to be true
end
```